sources/packages/web/src/services/ConfirmationOfEnrollmentService.ts

```diff
@@ -42,4 +47,20 @@ export class ConfirmationOfEnrollmentService {
       applicationId,
     );
   }
+
+  public async getCOEDenialReasons(): Promise<COEDeniedReasonDto> {
+    return ApiClient.ConfirmationOfEnrollment.getCOEDenialReasons();
+  }
+
+  public async denyConfirmationOfEnrollment(
+    locationId: number,
+    applicationId: number,
+    DenyCOEPayload: DenyConfirmationOfEnrollment,
```

Comment:
Please adjust the parameter to match the other letter cases: _denyCOEPayload_ instead of _DenyCOEPayload_.